components/user-mgt/org.wso2.carbon.user.mgt.workflow/src/main/java/org/wso2/carbon/user/mgt/workflow/userstore/AddRoleWFRequestHandler.java

```diff
@@ -97,8 +97,7 @@ public boolean startAddRoleFlow(String userStoreDomain, String role, String[] us
             fullyQualifiedName = UserCoreUtil.addDomainToName(userList[i], userStoreDomain);
             entities[i + 1] = new Entity(fullyQualifiedName, UserStoreWFConstants.ENTITY_TYPE_USER, tenant);
         }
-        if (workflowService.eventEngagedWithWorkflows(UserStoreWFConstants.ADD_ROLE_EVENT) && !Boolean.TRUE.equals
-                (getWorkFlowCompleted()) && !isValidOperation(entities)) {
+        if (!Boolean.TRUE.equals(getWorkFlowCompleted()) && !isValidOperation(entities)) {
```

Comment:
Please configure IDE to apply wso2 coding style.Seems there are alignment issues in some of PRs